app/scripts/models/account.js

```diff
@@ -1294,16 +1294,18 @@ define(function (require, exports, module) {
      * Verifies a TOTP code. If code is verified, token will be marked as verified.
      *
      * @param {String} code
-     *
+     * @param {Object} relier
      * @returns {Promise}
      */
-    verifyTotpCode (code) {
+    verifyTotpCode (code, relier) {
+      const options = {
+        metricsContext: this._metrics.getFlowEventMetadata(),
+        service: relier.get('service')
```

Comment:
Since only the `service` is used from the relier, any harm in just passing that info in instead of the full relier?